AngularLanguageService/AngularHtmlCompletionProvider.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+using Microsoft.VisualStudio.Utilities;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.WebTools.Languages.Html.Editor.Completion;
+using Microsoft.WebTools.Languages.Html.Editor.Completion.Def;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace AngularLanguageService
+{
+    [HtmlCompletionProvider("Children", "*")]
+    [ContentType(AngularTemplateContentDefinition.Name)]
+    public class AngularHtmlCompletionProvider : IHtmlCompletionListProvider
+    {
+        [Import]
+        private readonly ILanguageClientBroker broker = null;
+
+        public IList<HtmlCompletion> GetEntries(HtmlCompletionContext context)
+        {
+            var list = new List<HtmlCompletion>();
+
+            var angularCompletions = ThreadHelper.JoinableTaskFactory.Run(async delegate
+            {
+                JToken completions = await CallLanguageServiceBrokerAsync(context).ConfigureAwait(false);

```

Comment:
Should be `var` for consistency. Same for the other declarations in this file.

Reply:
Done!